packages/vow/src/types.ts

```diff
@@ -33,12 +33,7 @@ export type EVow<T> = ERef<T | Vow<T>>;
  * Follow the chain of vow shortening to the end, returning the final value.
  * This is used within E, so we must narrow the type to its remote form.
  */
-export type EUnwrap<T> =
-  T extends Vow<infer U>
-    ? EUnwrap<U>
-    : T extends PromiseLike<infer U>
-      ? EUnwrap<U>
-      : T;
+export type EUnwrap<T> = Fulfilled<T>;
```

Comment:
why didn't do do this before?

do we still need this type? perhaps it should be deprecated and direct users to `Fulfilled`?